spec/peep_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'peep'
+
+describe Peep do
+
+  describe '.all' do
+    it 'shows all peeps' do
```

Comment:
Its really great that you are doing unit tests as well as your feature tests! My only thought is that for unit tests it might be better to mock the database and only test the logic in your function. For example if you mocked the database to return a peep and then you can check that your map function works and returns the message part.